jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/saphana/SapHanaSqlDialect.java

```diff
@@ -0,0 +1,143 @@
+package com.exasol.adapter.dialects.saphana;
+
+import static com.exasol.adapter.AdapterProperties.*;
+import static com.exasol.adapter.capabilities.AggregateFunctionCapability.*;
+import static com.exasol.adapter.capabilities.AggregateFunctionCapability.VAR_SAMP;
+import static com.exasol.adapter.capabilities.LiteralCapability.*;
+import static com.exasol.adapter.capabilities.MainCapability.*;
+import static com.exasol.adapter.capabilities.PredicateCapability.*;
+import static com.exasol.adapter.capabilities.ScalarFunctionCapability.*;
+
+import java.sql.*;
+import java.util.*;
+
+import com.exasol.adapter.*;
+import com.exasol.adapter.capabilities.*;
+import com.exasol.adapter.dialects.*;
+import com.exasol.adapter.jdbc.*;
+import com.exasol.adapter.sql.*;
+
+/**
+ * This class implement the SAP HANA SQL dialect.
+ *
+ * @see <a href=
+ *      "https://help.sap.com/viewer/4fe29514fd584807ac9f2a04f6754767/2.0.03/en-US/f327b70cae564c53a766367a8aad0164.html">SAP
+ *      HANA SQL</a>
+ */
+public class SapHanaSqlDialect extends AbstractSqlDialect {
+    private static final String NAME = "SAPHANA";
+    private static final Capabilities CAPABILITIES = createCapabilityList();
+    private static final List<String> SUPPORTED_PROPERTIES = Arrays.asList(SQL_DIALECT_PROPERTY,
+            CONNECTION_NAME_PROPERTY, CONNECTION_STRING_PROPERTY, USERNAME_PROPERTY, PASSWORD_PROPERTY,
+            CATALOG_NAME_PROPERTY, SCHEMA_NAME_PROPERTY, TABLE_FILTER_PROPERTY, EXCLUDED_CAPABILITIES_PROPERTY,
+            DEBUG_ADDRESS_PROPERTY, LOG_LEVEL_PROPERTY);
+
+    /**
+     * Get the Sap Hana dialect name
```

Comment:
```suggestion
     * Get the SAP HANA dialect name
```

Please uniformly use the original letter capitalization of the company and product name in comments.

Reply:
fixed